packages/zoe/src/zoeService/zoe.js

```diff
@@ -197,7 +243,7 @@ const makeZoeKit = (
     makeZoeService(zoeServiceDataAccess),
   );
   zoeService = zoeServices.zoeService;
```

Comment:
I find the terminology here confusing. There are two levels of kit-ness. `vivifyFarClassKit` makes `makeZoeService` which doesn't make a ZoeService. It makes a `zoeServices` where the plural seems to stand in for the kit-ness.

The terminology I find confusing is from an earlier review I approved, where I should have flagged it. But I find it sufficiently confusing that I think it is worth changing anyway. OTOH, if you accept the suggestions at https://github.com/Agoric/agoric-sdk/pull/6735 , then the confusing terms go away, so we don't need to figure out what to call them ;)

Reply:
`zoeServices` was a quick wrapper to bundle the `zoeService`, (a kit with two facets) with `setVatAdminService`, which I thought ought to be distinct for separation of concerns reasons. I've almost talked myself out of that, but I'm not sure what the cleanest approach would be.